src/components/Alert.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement } from 'react';
+import { ExclamationCircleIcon } from '@heroicons/react/24/solid'
+
+/**
+ * Creates an alert with the given string message.
+ * 
+ * @returns The alert component
+ */
+const Alert = (props: {message: string, className?: string}): ReactElement =>  {
```

Comment:
I'm wondering what cases we might need to pass in arguments for `className`?

Reply:
Not sure I added it based on @melocule 's advice, but if it's extraneous we could remove it.